melodic/distribution.yaml

```diff
@@ -3843,11 +3843,11 @@ repositories:
       tags:
         release: release/melodic/{package}/{version}
       url: https://github.com/PilzDE/pilz_industrial_motion-release.git
-      version: 0.4.0-0
+      version: 0.4.1-0
     source:
       type: git
       url: https://github.com/PilzDE/pilz_industrial_motion.git
-      version: kinetic-devel
+      version: melodic-devel
```

Comment:
If we are changing the source branch here, should we also be changing the doc branch above?

Reply:
@clalancette Yes! Thanks for reading carefully.